File: python/1064_positivos_e_media.py
a = float(input())
b = float(input())
c = float(input())
d = float(input())
e = float(input())
f = float(input())

N = [a, b, c, d, e, f]

soma = 0
contador = 0

for num in N:
    if num > 0:
        contador += 1
        soma += num
        media = soma/contador

print(f'{contador} valores positivos')
print(f'{media:.1f}')

File: c/c99/1064_positivos_e_media.c
#include <stdio.h>

int main() {
    float a, b, c, d, e, f;
    float soma = 0, media;
    int contador = 0;

    scanf("%f", &a);
    scanf("%f", &b);
    scanf("%f", &c);
    scanf("%f", &d);
    scanf("%f", &e);
    scanf("%f", &f);

    float N[6] = {a, b, c, d, e, f};

    for (int i = 0; i < 6; i++) {
        if (N[i] > 0) {
            contador++;
            soma += N[i];
        }
    }

    media = soma/contador;

    printf("%d valores positivos\n", contador);
    printf("%.1f\n", media);

    return 0;
}
